composer.json

```diff
@@ -13,27 +13,24 @@
         {"name": "Ben Scholzen", "email": "bscholzen@soliantconsulting.com"}
     ],
     "require": {
-        "php": "^7.0",
-        "soliantconsulting/simplefm": "^5.0",
-        "container-interop/container-interop": "^1.1",
-        "zendframework/zend-diactoros": "^1.3",
-        "dasprid/treereader": "^1.2"
+        "php": "^7.2",
+        "dasprid/treereader": "^1.3",
+        "psr/container": "^1.0",
+        "soliantconsulting/simplefm": "^6.0.0-rc1",
```

Comment:
For the moment we are referencing the rc1 release here. Before tagging this with a new major version, make sure to reference a stable release.